src/snekmate/auth/Ownable2Step.vy

```diff
@@ -33,24 +45,15 @@ event OwnershipTransferStarted:
     new_owner: indexed(address)
 
 
-# @dev Emitted when the ownership is transferred
-# from `previous_owner` to `new_owner`.
-event OwnershipTransferred:
-    previous_owner: indexed(address)
-    new_owner: indexed(address)
-
-
 @deploy
 @payable
 def __init__():
     """
     @dev To omit the opcodes for checking the `msg.value`
          in the creation-time EVM bytecode, the constructor
          is declared as `payable`.
-    @notice The `owner` role will be assigned to
-            the `msg.sender`.
     """
-    self._transfer_ownership(msg.sender)
+    pass
```

Comment:
is this func needed? if yes, i would add a comment to clarify/make it clear that the ownership is set to the `msg.sender` by the `Ownable` module inherited.

Reply:
This function is needed since I have the opinionated design of making ctors `payable`.

> if yes, i would add a comment to clarify/make it clear that the ownership is set to the msg.sender by the Ownable module inherited.

Well, technically the ownership is not assigned here but in the contract where you initialise it. That's why I add a comment in the mock: https://github.com/pcaversaccio/snekmate/blob/refactor/Ownable2Step/src/snekmate/auth/mocks/OwnableMock.vy#L40-L41

But I think it doesn't harm to add a clarification here as well.

Reply:
gotcha. ++ on a clarification to initialize both -- if you are coming from oz contracts, it's easy to assume you only need to initialize `Ownable2Step`

Reply:
Added comment via https://github.com/pcaversaccio/snekmate/pull/219/commits/d5f20b7c29bc700a208f80baaea8279747cdd676. 

> if you are coming from oz contracts, it's easy to assume you only need to initialize `Ownable2Step`

Fortunately, the compiler will throw with an informative message & I have an example mock to showcases how to use it. If too many people complain about it, we can reconsider removing the `payable` approach.